pokedex/data/csv/pokemon_species_flavor_text.csv

```diff
@@ -63751,6 +63940,9 @@ many people detest it."
 いわれ　忌み嫌う　人も　多い。"
 198,30,12,"会为老大去寻找闪闪发光的东西。
 据说它很不吉利，厌恶它的人也很多。"
+198,39,9,"Widely shunned as a bearer of ill fortune. Upon crossing paths
+with this creature, I've been told one must chant "Workrum,
```

Comment:
Here the build complained about the use of `"`. They shouldn't be there. Maybe you can replace the double quotes with single ones.

Reply:
@Naramsim I have fixed, can you rebuild again?

Reply:
I'll try!

Reply:
```
building pokemon_v2_pokemonform
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.7/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.7/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.7/site-packages/django/core/management/base.py", line 316, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.7/site-packages/django/core/management/base.py", line 353, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python3.7/site-packages/django/core/management/commands/shell.py", line 92, in handle
    exec(sys.stdin.read())
  File "<string>", line 1, in <module>
  File "/code/data/v2/build.py", line 2272, in build_all
    _build_pokemons()
  File "/code/data/v2/build.py", line 1965, in _build_pokemons
    build_generic((PokemonForm,), "pokemon_forms.csv", csv_record_to_objects)
  File "/code/data/v2/build.py", line 92, in build_generic
    for obj in csv_record_to_objects(csv_record):
  File "/code/data/v2/build.py", line 1962, in csv_record_to_objects
    order=int(info[9]),
ValueError: invalid literal for int() with base 10: ''
```

The order has to be set, I guess. I remember that one could put 0.

Reply:
hi @Naramsim, I just added the order, it's increased from the last number

Reply:
Hi, you forgot to add the order to IDs 899->905.

Maybe you can add it, and shift the other `order`s you added previously

Reply:
Just added @Naramsim